<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisorImagenes</class>
 <widget class="QDialog" name="VisorImagenes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>331</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Cambria</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::East</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="icon">
       <iconset resource="../Recursos.qrc">
        <normaloff>:/Imagenes/iconos/add.png</normaloff>:/Imagenes/iconos/add.png</iconset>
      </attribute>
      <attribute name="title">
       <string/>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="4" column="1">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Nombre:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLineEdit" name="LineCarpeta">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Carpeta:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLineEdit" name="LineNombre">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QTreeWidget" name="VisorArbol">
           <column>
            <property name="text">
             <string>Imagenes</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Codigo</string>
            </property>
           </column>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="labelImagen">
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Codigo:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="LineCodigo">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="4" rowspan="3">
          <widget class="QPushButton" name="ButonSeleccion">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../Recursos.qrc">
             <normaloff>:/Imagenes/iconos/add.png</normaloff>:/Imagenes/iconos/add.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>45</width>
             <height>45</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="icon">
       <iconset resource="../Recursos.qrc">
        <normaloff>:/Imagenes/iconos/delete.png</normaloff>:/Imagenes/iconos/delete.png</iconset>
      </attribute>
      <attribute name="title">
       <string/>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
